JustTweak/Classes/Core/TweaksConfigurationsCoordinator.swift

```diff
@@ -127,14 +127,14 @@ final public class TweaksConfigurationsCoordinator: NSObject, TweaksConfiguratio
         return tweaks
     }
     
-    public func registerForConfigurationsUpdates(_ object: NSObject, closure: @escaping (_ tweakIdentifier: String?) -> Void) {
+    public func registerForConfigurationsUpdates(_ object: NSObject, closure: @escaping (Tweak) -> Void) {
         deregisterFromConfigurationsUpdates(object)
         let queue = OperationQueue.main
         let name = TweaksConfigurationDidChangeNotification
         let notificationsCenter = NotificationCenter.default
-        let observer = notificationsCenter.addObserver(forName: name, object: nil, queue: queue) { (notification) in
-            let tweakIdentifier = notification.userInfo?[TweaksConfigurationDidChangeNotificationTweakIdentifierKey] as? String
-            closure(tweakIdentifier)
+        let observer = notificationsCenter.addObserver(forName: name, object: nil, queue: queue) { notification in
+            guard let tweak = notification.userInfo?[TweaksConfigurationDidChangeNotificationTweakKey] as? Tweak else { return }
```

Comment:
This observer is listening to a notification that always contains the `Tweak` object. No need to have it optional.

Reply:
*Protective Mode ON* True, but if somebody else, for any reason send you the same notification without a Tweak object you crash the app *Protective Mode OFF*

Reply:
@federicocappelli it's guarding on that key, it won't crash the app